### ISSUE TITLE
Mobile Page mockup

### ISSUE THREAD

**Issue by mmaelicke** [hydrocode-de/integra-frontend#199]
@noobla11 before we share the URL of the application everywhere, we should implement a mock-up page, or a 'viewer application' (a lot of work) for the mobile variant of the application.

I think this is of high priority, as it has to happen soon, because a lot of people will follow links from a mobile device

**Comment by noobla11** [comment]
What exactly would a mock-up page look like and how time-consuming would this be? Do you mean just a page with an information that this page works only on laptop/pc?

**Comment by mmaelicke** [comment]
Yes, the minimal variant would be to inform the user that he should use the Desktop. That can be done pretty fast and I think we should go for that soon.

Once, we have a strategy, how data can be saved and shared (#176), we might also implement a page, that can only visualize the data, without editing. I think this would be a cool feature, that we can discuss **after** the 15.7. I can only work on that beforehand if you think that is an important feature.

**Comment by noobla11** [comment]
Sounds good! 

The visualising of the data in the mobile version would be a cool feature, but is more a "cherry on top" I would say, so rather low priority.

**Comment by noobla11** [comment]
I will think about a mock-up page desgn today or tomorrow and post it here.

**Comment by noobla11** [comment]
For now something like this might be enough, maybe we can also add a small slideshow or add pictures of the tool. 

![image](https://github.com/hydrocode-de/integra-frontend/assets/76469988/848407f1-34d1-4b0e-8a2c-dad72cccead3)

**Comment by mmaelicke** [comment]
Soll es genau dieses Bild werden? Haben wir kommerzielle Nutzungsrechte? Oder soll ich ein ähnliches Bild besorgen?

**Comment by noobla11** [comment]
Habe dir das Bild auf bwSync&Share hochgeladen unter dem Ordner "Bilder": https://bwsyncandshare.kit.edu/s/o8C6oEgADfttb7Z?path=%2FBilder
